src/actions/device.ts

```diff
@@ -93,12 +94,44 @@ export const savePresetToRemote = (name: string): AppThunk =>
 		oscQueryBridge.sendPacket(writePacket(message));
 	};
 
+export const loadPatcher = (name: string, inst: number = 0): AppThunk =>
+	() => {
+			let message = {
+				address: "/rnbo/inst/control/load",
+				args: [
+					{ type: "i", value: inst },
+					{ type: "s", value: name }
+				]
+			};
+			if (name === UNLOAD_PATCHER_NAME) {
+				message = {
+					address: "/rnbo/inst/control/unload",
+					args: [
+						{ type: "i", value: inst }
+					]
+				};
+			}
+			oscQueryBridge.sendPacket(writePacket(message));
+		};
+
+export const unloadPatcher = (inst: number = 0): AppThunk =>
```

Comment:
It doesn't look like this is ever actually used? Why do we have UNLOAD_PATCHER_NAME and unloadPatcher? Seems like they do the same thing?